pkgs/by-name/al/alphafold3/add-shebang.patch

```diff
@@ -0,0 +1,9 @@
+diff --git a/run_alphafold.py b/run_alphafold.py
+index 5e76b03..50eeda2 100644
+--- a/run_alphafold.py
++++ b/run_alphafold.py
+@@ -1,3 +1,4 @@
++#!/usr/bin/env python
```

Comment:
patchShebangs does this

Reply:
`patchShebangs` is only useful for shebangs that already exist.
There is no shebang in this target file.